examples/csharp/HelloworldUWP/GreeterServer/Protos/greet.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
use the same .proto file as the one used by other examples: `../../../protos/helloworld.proto`, no need to have a copy of the proto (and under a different name)

Reply:
Thanks, I'll fix that